easybuild/easyconfigs/m/medaka/medaka-1.4.3-foss-2020b.eb

```diff
@@ -0,0 +1,106 @@
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Danilo Gonzalez <danilo.gonzalez@hpcnow.com>
+# License::     GPL-v3.0
+# Updated to foss-2020b to use with artic tool
+# J. Sassmannshausen (GSTT/NHS UK)
+
+easyblock = 'PythonBundle'
+
+name = 'medaka'
+version = '1.4.3'
+
+homepage = 'https://github.com/nanoporetech/medaka'
+description = "medaka is a tool to create a consensus sequence from nanopore sequencing data."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+builddependencies = [('Autotools', '20200321')]
+
+dependencies = [
+    ('Python', '3.8.6'),  # includes cffi
+    # TensorFlow 2.2.3 required by medaka 1.4.3, see requirements.txt
+    # We need to patch medaka to use that version, see below. 
+    ('TensorFlow', '2.2.3'),
+    ('Biopython', '1.78'),
+    # Pysam 0.16.0.1 is strictly required, see requirements.txt
+    ('Pysam', '0.16.0.1'),
+    ('SAMtools', '1.12'),
+    # minimap2 is version 2.18 in pangolin, so we need the same version here
+    ('minimap2', '2.18'),
+    ('HTSlib', '1.11'),  # for tabix, bgzip
+    ('Racon', '1.4.21'),
+    ('spoa', '4.0.7'),
+    ('networkx', '2.6.2'),
+    ('edlib', '1.3.9'),
+    ('pyspoa', '0.0.8'),
+    ('pyfaidx', '0.5.9.5'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    # TF <= 2.4 needs h5py 2.10, later versions allow 3.x
+    ('h5py', '2.10.0', {
```

Comment:
I don't think we need to install `h5py` as an extension here, since it's part of the TensorFlow installation already?